new_aboutme/about_me2/header.php

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<!-- Required meta tags -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- Bootstrap CSS -->
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
+	<link rel="stylesheet" href="css/aboutme2.css">
+
+	<title></title>
+</head>
+
+<body>
+	<header class="container-fluid" id="header">
+		<h1>Vincent Ricci Jr - Graphic Designer</h1>
+		<nav class="navbar navbar-expand-lg navbar-light bg-light">
+			<div class="container-fluid">
+				<div class="collapse navbar-collapse" id="navbarSupportedContent">
+					<ul class="navbar-nav me-auto mb-2 mb-lg-0">
+						<li class="nav-item">
+							<a class="nav-link" aria-current="page" href="aboutme2.php">Home</a>
+						</li>
+						<li class="nav-item">
+							<a class="nav-link" href="podcasts.php">Knitting Podcasts</a>
+						</li>
+						<li class="nav-item dropdown">
+						<li class="nav-item">
+							<a class="nav-link" href="projects.php">Knitting Projects</a>
+						</li>
+						<li class="nav-item dropdown">
+						<li class="nav-item">
+							<a class="nav-link" href="dyeing.php">Yarn Dyeing</a>
+						</li>
+						<li class="nav-item dropdown">
+							<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Links</a>
+							<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+								<li><a class="dropdown-item" href="https://vincent-ricci-graphics.myportfolio.com/work" target="_blank">My Graphic Design Portfolio</a></li>
+								<li><a class="dropdown-item" href="https://mensknittingretreat.com/" target="_blank">Men's Knitting Retreats</a></li>
+								<li><a class="dropdown-item" href="https://buffaloknittingguild.com/" target="_blank">Knitting Guild of Greater Buffalo</a></li>
+								<li><a class="dropdown-item" href="https://imperialcourtofbuffalo.com/" target="_blank">Imperial Court of Buffalo</a></li>
+							</ul>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</nav><br />
+	</header>
+	<!-- Option 1: Bootstrap Bundle with Popper -->
+	<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+</body>
+
+</html>
```

Comment:
Don't need any of this either

Reply:
adjusted the header.php file and dropdown menu shows up now